docs/adr/009_lns_affinity.md

```diff
@@ -0,0 +1,286 @@
+# 009. LNS sticky affinity over multiple sessions
+
+**Feature**: [#1475](https://github.com/Azure/iotedge-lorawan-starterkit/issues/1475)  
+
+**Authors**:
+
+**Status**: Proposed
+__________
+
+## Problem statement
+
+Consider the topology:
+
+```mermaid
+flowchart LR;
+  Device-->LBS1-->LNS1--1-->IoTHub;
+  Device-->LBS2-->LNS2--2-->IoTHub;
+```
+
+where LNS 1 and 2 make use of their IoT Edge Hub modules to connect to IoT Hub.
```

Comment:
nit: they do use their module client, indeed, but I don't see why that is relevant when we want to talk about device connections? I would remove this, as it only adds confusion and distracts from the topic we want to describe.

Reply:
I had it here because the problem is more pronounced when using the Edge module but will remove since it doesn't go away completely even in direct mode.